### PR TITLE
Update eps-agenda.rst

### PR DESCRIPTION
**Pull request by JacalynLaird**
I thought I had the comms-edits branch open but apparently not so creating comms-edits-1 so I don't lose the changes!

### CODE REVIEW

File: docs/eog/eog-in-practice/cw21/event-project-stage/eps-agenda.rst [hunk 1]
```diff
@@ -17,12 +17,12 @@ Informing
...
 The keynote presentations were pre-recorded by the speakers.
 This was done so that the talks could be transcribed and captioned ahead of time, to mitigate risk of connection failures on the day, and also because both keynote speakers happened to be located in time zones incompatible with the event. 
-The length of each keynote session was 30 minutes, with the suggested format of up to 20 minutes for presentation and the remainder for introductions and questions. 
+The length of each keynote session was 30 minutes, with the suggested format of up to 20 minutes for the presentation and the remainder for introductions and questions. 
 
-The Chair of the session introduced the speaker live and reminded the participants that they could take notes and ask questions in the collaborative notes or via Sli.do, and that this session would be live streamed via YouTube. 
+The Chair of the session introduced the speaker live and reminded the participants that they could take notes and ask questions in the collaborative notes or via the app Sli.do, and that this session would be live streamed via YouTube.
```
Comment (shoaibsufi) [inline docs/eog/eog-in-practice/cw21/event-project-stage/eps-agenda.rst:25]:
```suggestion
The Chair of the session introduced the speaker live and reminded the participants that they could take notes and ask questions in the collaborative notes or via the Sli.do website/app, and that this session would be live streamed via YouTube. 
```

File: docs/eog/eog-in-practice/cw21/event-project-stage/eps-agenda.rst [hunk 2]
```diff
@@ -114,14 +114,14 @@ A schedule was scaffolded so that participants could work around suggested break
 
 Each Hack Day team had five minutes to demonstrate their projects and how they met the judging criteria:
 
-1. Novelty, creativity, coolness and/or usefulness
-2. Implementation and infrastructure 
-3. Demo and presentation
-4. Project transparency
-5. Future potential
-6. Team work
+1. Novelty, creativity, 'coolness' and/or usefulness.
```
Comment (shoaibsufi) [inline docs/eog/eog-in-practice/cw21/event-project-stage/eps-agenda.rst:117]:
```suggestion
1. Novelty, creativity, 'coolness' and/or usefulness
```

File: docs/eog/eog-in-practice/cw21/event-project-stage/eps-agenda.rst [hunk 3]
```diff
@@ -114,14 +114,14 @@ A schedule was scaffolded so that participants could work around suggested break
...
+2. Implementation and infrastructure.
```
Comment (shoaibsufi) [inline docs/eog/eog-in-practice/cw21/event-project-stage/eps-agenda.rst:118]:
```suggestion
2. Implementation and infrastructure 
```

File: docs/eog/eog-in-practice/cw21/event-project-stage/eps-agenda.rst [hunk 4]
```diff
@@ -114,14 +114,14 @@ A schedule was scaffolded so that participants could work around suggested break
...
 
-1. Novelty, creativity, coolness and/or usefulness
-2. Implementation and infrastructure 
-3. Demo and presentation
-4. Project transparency
-5. Future potential
-6. Team work
+1. Novelty, creativity, 'coolness' and/or usefulness.
+2. Implementation and infrastructure. 
+3. Demo and presentation.
```
Comment (shoaibsufi) [inline docs/eog/eog-in-practice/cw21/event-project-stage/eps-agenda.rst:119]:
```suggestion
3. Demo and presentation
```

File: docs/eog/eog-in-practice/cw21/event-project-stage/eps-agenda.rst [hunk 5]
```diff
@@ -114,14 +114,14 @@ A schedule was scaffolded so that participants could work around suggested break
...
+4. Project transparency.
```
Comment (shoaibsufi) [inline docs/eog/eog-in-practice/cw21/event-project-stage/eps-agenda.rst:120]:
```suggestion
4. Project transparency
```

File: docs/eog/eog-in-practice/cw21/event-project-stage/eps-agenda.rst [hunk 6]
```diff
@@ -114,14 +114,14 @@ A schedule was scaffolded so that participants could work around suggested break
...
+5. Future potential.
```
Comment (shoaibsufi) [inline docs/eog/eog-in-practice/cw21/event-project-stage/eps-agenda.rst:121]:
```suggestion
5. Future potential
```

File: docs/eog/eog-in-practice/cw21/event-project-stage/eps-agenda.rst [hunk 7]
```diff
@@ -114,14 +114,14 @@ A schedule was scaffolded so that participants could work around suggested break
...
+6. Team work.
```
Comment (shoaibsufi) [inline docs/eog/eog-in-practice/cw21/event-project-stage/eps-agenda.rst:122]:
```suggestion
6. Team work
```

Comment (shoaibsufi) [reply docs/eog/eog-in-practice/cw21/event-project-stage/eps-agenda.rst:122]: Again due to - https://github.com/softwaresaved/event-organisation-guide/blob/master/STYLE.md#bulleted-lists-punctuation - we consistently have followed this so changing would mean changing a lot of other parts of EOG

**Review by shoaibsufi — CHANGES_REQUESTED** [review]
Just some minor changes around bulleted list punctuation style in EOG - https://github.com/softwaresaved/event-organisation-guide/blob/master/STYLE.md#bulleted-lists-punctuation

**Comment by JacalynLaird** [comment]
ready for review @shoaibsufi

**Review by shoaibsufi — APPROVED** [review]
Looks good

**Pull request merged**